src/status_im2/contexts/quo_preview/wallet/confirmation_progress.cljs

```diff
@@ -0,0 +1,80 @@
+(ns status-im2.contexts.quo-preview.wallet.confirmation-progress
+  (:require
+    [quo.core :as quo]
+    [react-native.core :as rn]
+    [reagent.core :as reagent]
+    [status-im2.contexts.quo-preview.preview :as preview]))
+
+(def descriptor
+  [{:type :text
+    :key  :progress-value}
+   {:type    :select
+    :key     :network
+    :options [{:key :mainnet}
+              {:key :optimism}
+              {:key :arbitrum}]}
+   {:type    :select
+    :key     :state
+    :options [{:key :pending}
+              {:key :sending}
+              {:key :confirmed}
+              {:key :finalising}
+              {:key :finalized}
+              {:key :error}]}
+   (preview/customization-color-option)])
+
+(def total-box 85)
+(def counter (reagent/atom 0))
+(def interval-id (reagent/atom nil))
+(def interval-ms 50)
+
+(defn stop-interval
+  []
+  (when @interval-id
+    (js/clearInterval @interval-id)
+    (reset! interval-id nil)))
+
+(defn clear-counter
+  []
+  (reset! counter 0))
+
+(defn update-counter
+  [state]
+  (let [new-counter-value (-> @counter inc)]
```

Comment:
I think using threading macros make sense when we have more than one step, for this one, I think it's better to just use `(inc @counter)`.

Reply:
Done